tests/integration/base_file_test.go

```diff
@@ -124,7 +135,7 @@ spec:
   volumes:
   - name: ` + filesystemName + `
     flexVolume:
-      driver: rook.io/rook
+      driver: rook.io/` + driverName + `
```

Comment:
looks like my final comment wasn't saved.  Is this saying that to use CephFS volumes, you have to use a dynamic driver name? if the user uses a namespace other than `rook-system` for the operator/agent, they would have to change this value? that doesn't feel right, but maybe i'm not understanding.

Reply:
you can still use `rook.io/rook` for the driver name since we start a driver for both the namespace name and the static `rook` name. However, you could hit the conflict, in particular this test was hitting the conflict.